### DISCUSSION TITLE
TLS certificates renewal support

### DISCUSSION THREAD

**Discussion by antonio-oliva** [edenhill/librdkafka#4490] [Q&A]
Hi all,
I do not know if this is the right place to ask this question. I apologize in advance if it is not.
I was wondering if the latest version of the confluent-kafka-go library, i.e., v2.3.0, supports certificates renewal when TLS is used for the broker-clients communication. 
The scenario I would like to implement is:
1. Setting up the TLS configuration with the proper certificate
2. Every time the old certificates are substituted with renewed ones (even before the expiration) broker and clients to automatically reload certificates, without having to restart them.

Is such a behavior implemented in confluent-kafka-go v2.3.0?

**Comment by danesss** [comment]
FYI, this open issue discusses a similar scenario: https://github.com/confluentinc/librdkafka/issues/2868
